part-2/2.2_setting_up_your_load_balancer.md

```diff
@@ -1,12 +1,15 @@
 # Load Balancer 
 A managed AWS service that distributes traffic to targets you specify
 
-There are 3 types:
+There are 4 types:
 - ALB - HTTP/S (layer 7)
 - NLB - TCP, UDP (layer 4)
-- ELB - Basic load balancing for EC2 👈🏿 This is what we're going to work with!
+- GLB - To manage a fleet of third-party virtual appliances that support GENEVE
+- ELB -(TCP, SSL/TLS, HTTP, HTTPS (Layer 4 and 7) Basic load balancing for EC2 👈🏿 This is what we're going to work with!
 
-![LB Diagram](images/elb_network_diagram.png)
+[Here's the Docs on their differences in more detail](https://aws.amazon.com/elasticloadbalancing/features/#Product_comparisons)
+
+![LB Diagram](2022-images/3-elb-network-diagram.png)
```

Comment:
I could be wrong, but I'm not sure this one is correct as it's not loading in the Github preview

Reply:
nevermind, i didnt realise rich diff was turned on file by file

Reply:
that one is just a link :)